frontend/app/routes/$lang+/_protected+/dependent-status-checker+/index.tsx

```diff
@@ -0,0 +1,117 @@
+import type { LoaderFunctionArgs, MetaFunction } from '@remix-run/node';
+import { json } from '@remix-run/node';
+import { useLoaderData } from '@remix-run/react';
+
+import { useTranslation } from 'react-i18next';
+
+import pageIds from '../../page-ids.json';
+import { ContextualAlert } from '~/components/contextual-alert';
+import { getAuditService } from '~/services/audit-service.server';
+import { getInstrumentationService } from '~/services/instrumentation-service.server';
+import { getLookupService } from '~/services/lookup-service.server';
+import { getRaoidcService } from '~/services/raoidc-service.server';
+import { featureEnabled, getEnv } from '~/utils/env.server';
+import { getNameByLanguage, getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { getFixedT } from '~/utils/locale-utils.server';
+import { mergeMeta } from '~/utils/meta-utils';
+import { IdToken, UserinfoToken } from '~/utils/raoidc-utils.server';
+import { RouteHandleData } from '~/utils/route-utils';
+import { getTitleMetaTags } from '~/utils/seo-utils';
+
+export const handle = {
+  i18nNamespaces: getTypedI18nNamespaces('dependent-status-checker', 'gcweb'),
+  pageIdentifier: pageIds.protected['dependent-status-checker'].index,
+  pageTitleI18nKey: 'dependent-status-checker:page-title',
+} as const satisfies RouteHandleData;
+
+export const meta: MetaFunction<typeof loader> = mergeMeta(({ data }) => {
+  return data ? getTitleMetaTags(data.meta.title) : [];
+});
+
+export async function loader({ context: { session }, params, request }: LoaderFunctionArgs) {
+  featureEnabled('dependent-status-checker');
+
+  const raoidcService = await getRaoidcService();
+  await raoidcService.handleSessionValidation(request, session);
+  const instrumentationService = getInstrumentationService();
+
+  const idToken: IdToken = session.get('idToken');
+  getAuditService().audit('page-view.status-check', { userId: idToken.sub });
```

Comment:
```suggestion
  getAuditService().audit('page-view.dependent-status-checker', { userId: idToken.sub });
```

Reply:
Thanks for catching that. Implemented in commit ade45087bcec0267b9518c0e361aae2000367100.

Reply:
`dependent-status-checker` and not `dependent-status-check`

Reply:
Fixed in commit 48b8a0c4a1b5742aa8d72ae86f64379a7096e38a